internal/config/stages.go

```diff
@@ -29,6 +29,12 @@ type hclBuild struct {
 	LabelScoped []*scopedStage `hcl:"label,block"`
 }
 
+type hclStep struct {
```

Comment:
I think this should go in `pipeline.go`

Reply:
Agreed! Got that moved in 1f12ea6923c56433eb71c547e1f1a57e22a40bb4